src/handlers/shared/start.ts

```diff
@@ -48,7 +48,7 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
 
   // check for max and enforce max
 
-  if (assignedIssues.length - openedPullRequests.length >= maxTask) {
+  if (assignedIssues.length - openedPullRequests.length >= 3) {
```

Comment:
this should be something like `maxTasks.limit` or however you decide to name the `getUserRole()` return var